### PR TITLE
Added Monetalis Clydesdale overview description to collateral pages.

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: collateral/collateral-overview.md [hunk 1]
```diff
@@ -155,6 +155,10 @@ A variety of real estate, business, and consumer loans with a range of durations
 
 [More info (Huntingdon Valley Bank)...](hvbank.md)
 
+## Societe Generale
+> **Ilks: ** RWA008-A
+
+Different blends of short-terms bonds from ultra-short (<1 year) to short (1-3 years) maturities. Investments done through BlackRock and Baillie Gifford. Mainly invested in US Treasuries but some exposure to corporate bonds.
```
Comment (patrick-j-govalpha) [inline collateral/collateral-overview.md:161]:
```suggestion
Different blends of short-term bonds from ultra-short (<1 year) to short (1-3 years) maturities. Investments done through BlackRock and Baillie Gifford. Mainly invested in US Treasuries but some exposure to corporate bonds.
```

Comment (patrick-j-govalpha) [inline collateral/collateral-overview.md:161]: Can you point to where the Blackrock and Baillie Gifford information is? Just tried ctrl+f on the socgen proposal threads and can't find those brokers listed.

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: collateral/collateral-overview.md (hunk 1 above)
Comment (LongForWisdom) [reply collateral/collateral-overview.md:161]: Nope, CES wrote it, not sure what source they're using.

Comment (patrick-j-govalpha) [reply collateral/collateral-overview.md:161]: Hmm. I'm suspicious that someone has mixed up SocGen and Monetalis. The SocGen application states:

"the OFH Tokens are characterized as covered bonds under French Law backed by home loans and benefiting from a statutory privilege, issued by Societe Generale SFH (SG SFH) a specialized credit institution with the status of Société de Financement de l’Habitat delivered by the Authorité de Contrôle Prudentiel (as further described below) in the form of security tokens in the Ethereum public blockchain."

which doesn't sounds like the description above

Comment (LongForWisdom) [reply collateral/collateral-overview.md]: You're correct, it's supposed to apply to Monetalis. My fault, I think, was reading discord chat wrong. Embarassing. We don't even have Monetalis onboard yet :/ Will update and can merge when Monetalis draws funds.

**Review by patrick-j-govalpha — APPROVED** [review]

**Pull request merged**